gui/wxpython/modules/import_export.py

```diff
@@ -116,14 +116,12 @@ def __init__(self, parent, giface, itype,
         # buttons
         #
         # cancel
-        self.btn_close = Button(parent=self.panel, id=wx.ID_CLOSE)
+        self.btn_close = CancelButton(parent=self.panel)
         self.btn_close.SetToolTip(_("Close dialog"))
```

Comment:
This changes is from "Close" to "Cancel", I would think "Close" would be often more appropriate. Maybe we need a CloseButton? This is on other places as well.

Reply:
Maybe a bad move for me to mix addressing the original issue (with key bindings) with GUI changes from `Close` to `Cancel`!

This PR was intended to be a step in (and prerequisite for) a more general reorganisation of buttons as I mentioned earlier https://github.com/OSGeo/grass/pull/744#issuecomment-663977673. The rationale for the change of labels is to make a consistent GUI: both within GRASS and compared to common practice in various applications and OSs. Now with GRASS the two labels are used variably. In my experience, in most environments you **close** a document/file, but **cancel** a dialog window.

Reply:
> In my experience, in most environments you **close** a document/file, but **cancel** a dialog window.

I think you *close* a document/file/book and *cancel* an operation. If there is an operation behind a dialog, let's say import, Cancel button should cancel the import operation. If the dialog is used to run an operation (import file), but is open even when the operation was started (or finished; file was imported), Close button is better since you are closing the dialog/window, nothing is being canceled (the operation was done already; file was imported). Perhaps the issue is window versus dialog terminology or perhaps you are already using this rule (I didn't check).

Reply:
I find the "by opening dialog window you already started an operation" part convincing. I'll think about it more for the Cancel/Close PR.